app/dependencies.php

```diff
@@ -391,5 +392,19 @@ function (ContainerInterface $c): Auth\DonationPublicAuthMiddleware {
                 new PhpSerializer(),
             );
         },
+        Connection::class => static function (ContainerInterface $c): Connection {
+            $em = $c->get(EntityManagerInterface::class);
+            \assert($em instanceof EntityManagerInterface);
```

Comment:
getting quite bored of all these asserts, think I'll put a Psalm stub for the ContainerInterface in next time that says that it always `T` when we pass `class-string<T>`